app/models/discussion.rb

```diff
@@ -129,8 +137,10 @@ def delayed_destroy
   end
 
   def thread_item_created!(item)
-    self.items_count += 1
-    self.last_item_at = item.created_at
+    if THREAD_ITEM_KINDS.include? item.kind
```

Comment:
I've been thinking about ripping this out and replacing it with https://github.com/cedric/custom_counter_cache